### DISCUSSION TITLE
Italics?

### DISCUSSION THREAD

**Discussion by be5invis** [orioncactus/pretendard#132] [Q&A] [answered]
It might be helpful to introduce italics with proper slope angle and proper Latin variant glyphs.

**Comment by orioncactus** [comment, marked as answer]
Hello! Unfortunately, Pretendard Std(Latin/Greek/Cyrillic-only version of Pretendard) is a derivative version of Pretendard that doesn't support Italic, and since Pretendard itself has no plans to support Italic, there are currently no plans to support Italic in Pretendard Std either. Could you please let me know of any situations where Italic is needed? Understanding the frequency of Italic usage could be helpful in developing Italic.

**Comment by hyeshik** [comment]
I second to request the support of italic variants. In Korean academic writings, the jargons are often written in English or Latin rather than Korean. Still, the English or Korean words are often attached to Korean postpositions. Therefore, it is very tricky to keep consistency in baseline, letter heights, line height when using Pretendard as a main font and change the font temporarily to other latin fonts with italic variants. Here are a typical example:

_M. musculus_ 의 fibroblast에서 뽑은 total RNA에서 정량한 결과 _TRIM98_ 이 knockdown되었을 때 _Gapdh_ 에 비교해서 _Eif3a_ 의 발현양이 크게 감소하였다.

Since the academic writings require somewhat strict conformation to the standard nomenclature, it is particularly important to correctly italicize the words as intended. In some cases, an italicized word could have different meaning than a upright version. For example,

1. _ZCCHC44_ 가 감소하였다.
2. ZCCHC44 가 감소하였다.

Those two expressions have different meanings. In human biology contexts, (1) means the messenger RNA or gene expression is decreased, but (2) strictly states that the protein is decreased.

It would be truly great if Pretendard adds a support for italic fonts at least for latin letters.
